app/assets/javascripts/admin/components/admin-web-hook-event-chooser.js.es6

```diff
@@ -0,0 +1,46 @@
+import computed from 'ember-addons/ember-computed-decorators';
+
+export default Ember.Component.extend({
+  tagName: '',
+  typeName: Ember.computed.alias('type.name'),
+
+  @computed('typeName')
+  name(typeName) {
+    return I18n.t(`admin.web_hooks.${typeName}_event.name`);
+  },
+
+  @computed('typeName')
+  details(typeName) {
+    return I18n.t(`admin.web_hooks.${typeName}_event.details`);
+  },
+
+  @computed
+  eventTypeExists: {
+    get() {
```

Comment:
Nitpick: If we're not using `set()`, we should follow the rest of the convention and just define the computed property without `get()`